Description: Don't build libraries
 Undo factoring out of the libraries for the time being as they do not
 appear to be used anywhere. There might be the possibility that the
 libraries are moved to another upstream and this would complicate
 packaging if the libraries are built.
Author: Andreas Bombe <aeb@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: media-ctl/Makefile.am
===================================================================
--- media-ctl.orig/Makefile.am	2013-01-19 15:50:13.094776981 +0100
+++ media-ctl/Makefile.am	2013-01-31 01:07:33.466485594 +0100
@@ -1,5 +1,2 @@
 SUBDIRS = src
 ACLOCAL_AMFLAGS = -I m4
-pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = libmediactl.pc libv4l2subdev.pc
-
Index: media-ctl/configure.ac
===================================================================
--- media-ctl.orig/configure.ac	2013-01-19 15:50:13.094776981 +0100
+++ media-ctl/configure.ac	2013-01-31 01:07:25.107590095 +0100
@@ -85,8 +85,6 @@
 AC_CONFIG_FILES([
  Makefile
  src/Makefile
- libmediactl.pc
- libv4l2subdev.pc
 ])
 
 AC_OUTPUT
Index: media-ctl/src/Makefile.am
===================================================================
--- media-ctl.orig/src/Makefile.am	2013-01-19 15:50:13.095776969 +0100
+++ media-ctl/src/Makefile.am	2013-01-31 01:10:04.627595820 +0100
@@ -1,12 +1,5 @@
-lib_LTLIBRARIES = libmediactl.la libv4l2subdev.la
-libmediactl_la_SOURCES = mediactl.c
-libv4l2subdev_la_SOURCES = v4l2subdev.c
-mediactl_includedir=$(includedir)/mediactl
-mediactl_include_HEADERS = mediactl.h v4l2subdev.h
-
 bin_PROGRAMS = media-ctl
 media_ctl_CFLAGS = $(LIBUDEV_CFLAGS)
 media_ctl_LDFLAGS = $(LIBUDEV_LIBS)
-media_ctl_SOURCES = main.c options.c options.h tools.h
-media_ctl_LDADD = libmediactl.la libv4l2subdev.la
-
+media_ctl_SOURCES = main.c options.c options.h tools.h \
+		   v4l2subdev.c v4l2subdev.h mediactl.c mediactl.h
